### PR TITLE
Fix: Withdrawal border _get_withdrawal_request_status calls now getWithdrawalStatus method

### PR DESCRIPTION
**Pull request by F4ever**

### CODE REVIEW

File: src/web3py/extensions/tx_utils.py
Comment (skhomuti) [inline src/web3py/extensions/tx_utils.py:1]: it should be in a separate PR

Comment (F4ever) [reply src/web3py/extensions/tx_utils.py:1]: Fixed
